website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -6,12 +6,12 @@
   </tr>
 </table>
 
-
```

Comment:
Please don't delete this line. I think having two empty lines in front of headers helps separate them visually.